### PR TITLE
Filecoin specific explore page, network picker on mainnet, network picker tweaks

### LINKED ISSUE #1241: New Explore page
The Explore page will be basically empty on the Filecoin deployment of the app. Let's put something else here.

image

### LINKED ISSUE #1309: Display network picker prominently even while wallet connected
To combat people being confused about what network they're on, let's display the network picker prominently next to the address also while already connected to a wallet. It should still be hidden under the account menu on mobile, where we don't have any space left in the top bar.

### LINKED ISSUE #1316: Enable Network Picker on Mainnet deployment

### PR DESCRIPTION
**Pull request by efstajas**
Screenshot 2024-10-28 at 13 02 10

Adds simple plumbing for showing different Explore pages depending on network, adds a light Filecoin-specific explore page, makes it so that the Network picker is never hidden in account menu, and enables the Network Picker on mainnet. 

Resolves #1309 
Resolves #1241
Resolves #1316

### CODE REVIEW

File: src/routes/app/(app)/+page.server.ts [hunk 1]
```diff
@@ -1,141 +1,20 @@
...
-        ... on ClaimedProjectData {
-          chain
-        }
-        ... on UnClaimedProjectData {
-          chain
-        }
-      }
+export const load = async ({ fetch }) => {
+  switch (network.chainId) {
+    case 314: {
```
Comment (mhgbrown) [inline src/routes/app/(app)/+page.server.ts:7]: Worth constantizing with a variable?

File: src/routes/app/(app)/components/default-explore-page.svelte [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+  $: featuredProjects = projects.filter((p) =>
+    FEATURED_PROJECT_ACCOUNT_IDS.includes(p.account.accountId),
+  );
+
+  // 2 latest posts. Sort by date
+  $: blogPosts = blogPosts
+    .sort((a, b) => new Date(b.date).getTime() - new Date(a.date).getTime())
+    .slice(0, 2);
+
+  // Last 4 projects. TODO: sort by claim date
```
Comment (mhgbrown) [inline src/routes/app/(app)/components/default-explore-page.svelte:102]: Just noting you have a TODO here. Worth turning into a Github issue?

File: src/routes/app/(app)/components/default-explore-page.svelte [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+  $: recentlyClaimedProjects = projects.slice(-4);
+
+  function numberWithCommas(input: number) {
```
Comment (mhgbrown) [inline src/routes/app/(app)/components/default-explore-page.svelte:105]: This looks like it's used to display a currency value. How about using `Intl`? 
`new Intl.NumberFormat('en-US', { style: 'currency', currency: 'USD' }).format(123567)`

File: src/routes/app/(app)/components/load-default-explore-page-data.ts [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+import {
+  ProjectSortField,
+  ProjectVerificationStatus,
+  SortDirection,
+} from '$lib/graphql/__generated__/base-types';
+import network from '$lib/stores/wallet/network';
+
+const FEATURED_DRIP_LISTS =
+  {
+    1: [
```
Comment (mhgbrown) [inline src/routes/app/(app)/components/load-default-explore-page-data.ts:29]: What do 1 and 5 mean is this context? I notice you used them in an earlier file too.

File: src/routes/app/(app)/components/default-explore-page.svelte [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+    {
+      1: [
+        '80921576051643469277397866636792942368647018452892810554457309839360',
+        '80928956806149918791864723629668437820661066502202314166815319654400',
+        '80989205010981758696261160004449877944077887004065826078532843448906',
+        '80921140646830818724035150101819719966329403614944137690624336855040',
+        '80912096692731427285642748238291568975884076216206760523486136893440', // nice-node
+        '80927335273972468167722947750338907267861671542981060844246982983680', // starknet.js
+        '80927325632295926773992520689210905998818993360029926329589912567808', // snapshot
+        '80922395546375089598655709477693009806793075640399849243804470083584', // graph-node
```
Comment (mhgbrown) [inline src/routes/app/(app)/components/default-explore-page.svelte:73]: These comments are helpful!

**Review by mhgbrown — APPROVED** [review]
Let's go Filecoin! I left some clarifying questions/suggestions.

File: src/routes/app/(app)/components/default-explore-page.svelte (hunk 2 above)
Comment (efstajas) [reply src/routes/app/(app)/components/default-explore-page.svelte:102]: ah yeah i just copied all this from the existing explore page code. Don't think this TODO actually applies anymore, will remove it

File: src/routes/app/(app)/components/default-explore-page.svelte (hunk 3 above)
Comment (efstajas) [reply src/routes/app/(app)/components/default-explore-page.svelte:105]: yeah def. also just copied it but good opportunity to clean it up, will do

File: src/routes/app/(app)/components/load-default-explore-page-data.ts (hunk 4 above)
Comment (efstajas) [reply src/routes/app/(app)/components/load-default-explore-page-data.ts:29]: those are chain IDs — I will add explicit typing to that var so it's clear

**Pull request merged**